### PR TITLE
Komikku: update to 0.15.0.

### PR DESCRIPTION
**Pull request by not-chicken**

### CODE REVIEW

**Comment by not-chicken** [comment]
Build errored on i686 because travis failed to fetch repodata and then reached max log length while building the dependencies from scratch. : /

File: srcpkgs/Komikku/template [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Template file for 'Komikku'
 pkgname=Komikku
-version=0.14.0
+version=0.15.0
 revision=1
 archs=noarch
-wrksrc=Komikku-v${version}
+wrksrc=${pkgname}-v${version}
```
Comment (Chocimier) [inline srcpkgs/Komikku/template:6]: Why is this better?

Comment (not-chicken) [reply srcpkgs/Komikku/template:6]: It isn't. I just did to be consistent with other packages in the repo.

Comment (Chocimier) [reply srcpkgs/Komikku/template:6]: There are packages that use literal too.
Using $pkgname is not semantic in wrksrc, if we create variant of package, say Komikku-lts (not probable for app, but happens for libraries or databases and so on), it is still `Komikku` needed here.

Comment (not-chicken) [reply srcpkgs/Komikku/template]: Alright my bad. I have changed it back. I'll keep this in mind from now on.

**Pull request merged**